test/integration/model/json.test.js

```diff
@@ -243,6 +243,23 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         });
       });
 
+      it('should be possible to save/update a NULL as the field value', function () {
+        var that = this;
+        return that.Event.create({data:{a:1}}) // create with non-NULL data field
+        .then( function(event) { // set data field to null
+          event.data = null;
+          return event.save();
+        })
+        .then( function(event) { // fetch it again
+          return that.Event.findAll({id: event.id});
+        })
+        .then( function(events) { // asserts
+          expect(events.length).to.equal(1);
+          var new_event = events[0];
+          expect(new_event.get('data').to.eql(null));
```

Comment:
The bracket is not set right. It should be `expect(new_event.get('data')).to.eql(null);`